### PR TITLE
Add VALET

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

This PR adds VALET, a tool to detect mis-assemblies in metagenomic assemblies.

Berenice

### CODE REVIEW

**Comment by martenson** [comment]
can we trim those 200k lines test files?

**Comment by bebatut** [comment]
I took the test data from the tool. I can try to trim them but it is not straightforward (will require quite a lot of steps to be sure to conserve meaningful reads)

File: tools/valet/valet.xml [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+        #set sep = ''
+        #set format = ''
+        #for $repeat in $input_reads.paired_collection_reads
+            #set $mate_1 += $sep + str($repeat.input.forward)
+            #set $mate_2 += $sep + str($repeat.input.reverse)
+            #set $minins += $sep + str($repeat.minins)
+            #set $maxins += $sep + str($repeat.maxins)
+            #set $sep = ","
+            #if $repeat.input.forward.is_of_type('fasta')
+                #set $format = '--fasta'
```
[the remark this answers is not in the document]
Comment (bebatut) [reply tools/valet/valet.xml]: I do not think so. Do you think I should split it (adding a selector to choose the format and then adding the data param)?

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/valet/valet.xml]: I guess this would be more save.

**Comment by bgruening** [comment]
Thanks @bebatut!

**Comment by martenson** [comment]
@bebatut much appreciated! 💯

**Pull request merged**